model-optimizer/extensions/front/onnx/resize_to_interpolate.py

```diff
@@ -26,7 +26,7 @@
 
 
 class ResizeToInterpolate2D(FrontReplacementSubgraph):
-    enabled = True
+    enabled = False
```

Comment:
Can we remove these transformations completely?

Reply:
Done.